mne/report.py

```diff
@@ -1184,7 +1184,7 @@ class construction.
                                             slider_id=slider_id, html=html,
                                             image_html=image_html))
 
-        self.fnames.append('%s-#-%s-#-custom' % (section, sectionvar))
+        self.fnames.append('%s-#-%s-#-custom' % (title, sectionvar))
```

Comment:
shouldn't this be: 
```py
self.fnames.append('%s-#-%s-#-custom' % (caption, sectionvar))
```
?

Reply:
no, sliders have a `caption` for each frame, but the `title` is unique.